src/components/pill-toggle/__definition__.js

```diff
@@ -0,0 +1,43 @@
+import pill-toggle from './';
```

Comment:
What is this definition for `pill-toggle`?

Reply:
Sorry this was left in from previous implementation. I'll remove it